libvirt/install_operator.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+#
+# Copyright Confidential Containers Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Install the CAA operator in a Kubernetes cluster.
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+LIBVIRT_IP="${LIBVIRT_IP:-}"
+LIBVIRT_USER="${LIBVIRT_USER:-$USER}"
```

Comment:
commit msg mentions default user is `wmoschet`, which is very cool name, it was disappointing to discover it used some other lame name on my system ;)  I'd update the commit msg so other won't be disappointed as well

Reply:
lol